share/spack/gitlab/cloud_pipelines/configs/ci.yaml

```diff
@@ -22,6 +22,10 @@ ci:
       script::
       - spack --color=always --backtrace ci rebuild --tests > >(tee ${SPACK_ARTIFACTS_ROOT}/user_data/pipeline_out.txt) 2> >(tee ${SPACK_ARTIFACTS_ROOT}/user_data/pipeline_err.txt >&2)
       after_script:
+      - - spack python ${CI_PROJECT_DIR}/share/spack/gitlab/cloud_pipelines/scripts/common/aggregate_package_logs.spack.py
+          --prefix /home/software/spack:${CI_PROJECT_DIR}
+          --log install_times.json
+          ${SPACK_ARTIFACTS_ROOT}/install_times.json
```

Comment:
Also, this artifact is ignored.  Could it go into `${SPACK_ARTIFACTS_ROOT}/logs/`?  We don't pick up as artifact just anything that appears in `$SPACK_ARTIFACTS_ROOT`, but only `logs/`, `reproduction\`, `tests\`, and `user_data\`.

Reply:
yeah, I think I fixed that in a wip PR for testing but removed it during a rebase.